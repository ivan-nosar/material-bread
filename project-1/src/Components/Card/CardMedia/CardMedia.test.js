import React from 'react';
import CardMedia from './CardMedia';

import renderer from 'react-test-renderer';

test('CardMedia Renders', () => {
  const tree = renderer.create(<CardMedia />).toJSON();
  expect(tree).toMatchSnapshot();
});
