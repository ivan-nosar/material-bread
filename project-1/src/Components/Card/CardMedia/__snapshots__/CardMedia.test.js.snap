// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardMedia Renders 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "overflow": "hidden",
        "position": "relative",
        "width": "100%",
      },
      Object {
        "height": 194,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "bottom": 16,
        "left": 16,
        "position": "absolute",
      }
    }
  />
</View>
`;
